modules/install-upgrade/pages/production-cluster-multiregion.adoc

```diff
@@ -0,0 +1,134 @@
+:activeTopics: 1
+:messageSize: 1000
+:messageThroughput: 100000
+:retentionPolicy: 3600
+:ttlPolicy: 24
+:tieredStoragePolicy: n/a
+:messageReplicationFactor: 3
+:clusterReplicationFactor: 2
+
+= Multi-region georeplication
+
+This page contains sizing and configuration information for a georeplicated VM-based production cluster. +
+Pulsar clusters can be configured to replicate data across multiple regions to maximize availability.
+
+We apply our sizing methodology with the following assumptions: +
+
+* Replication will be *asynchronous* and *will not* use a global Zookeeper.
+* We will replicate *all topics* across two regions.
+* We will assume the same time-to-live and retention policies as the initial Pulsar cluster (the cluster being replicated from).
+
+[#aggregate-worksheet]
```

Comment:
Generally speaking, I really don't think this calculation is needed.
As long as we have the calculation of one cluster (in another doc), the sizing for a geo-replication case is simply times the number of clusters. Repeating the same calculation step is not necessary.